phpBB/styles/prosilver/template/ucp_notifications.html

```diff
@@ -7,38 +7,6 @@ <h2 class="cp-title">{TITLE}</h2>
 	<div class="inner">
 
 		<p class="cp-desc">{TITLE_EXPLAIN}</p>
-
-		<!-- IF MODE == 'notification_options' -->
```

Comment:
Why did you remove this block?

Reply:
Has been moved to ucp_notifications_options.html which is used instead of this template file